src/DynamoCoreWpf/UI/GuidedTour/CustomRichTextBox.cs

```diff
@@ -171,8 +171,8 @@ private static FlowDocument GetCustomDocument(string Text)
                         imageFig.Height = new FigureLength(30);
                         para.Inlines.Add(imageFig);
                     }
-                    //End fo the Bullet Items formatting
-                    if (word.EndsWith("-"))
+                    //End of the Bullet Items formatting
```

Comment:
honestly, I'm not sure what this comment is trying to get across but I think it's supposed to be `of` - @RobertGlobant20 any hints?

Reply:
Well is referring that if you find the next text inside the CustomRichTextBox
`- Bullet number 1 asdfasdfasdfasdfasdfasdfasfdasdfasfasfdasfasfdasdfasfdas -`
it will put a bullet at the beginning and is expecting the "-" at the end so we will know which text should be included.
yes, I agree it should be "of"